integrations/enumeratum/src/test/scala/sttp/tapir/codec/enumeratum/TapirCodecEnumeratumTest.scala

```diff
@@ -77,7 +77,7 @@ class TapirCodecEnumeratumTest extends AnyFlatSpec with Matchers {
     }
   }
 
-  private def fullName[E](e: E) = s"$className${e.getClass.getSimpleName}".replace("$", ".")
+  private def fullName[E](e: E) = s"$className${e.getClass.getSimpleName}".replace("$", ".").dropRight(1)
```

Comment:
maybe we should spell out the expected name explicitly (as a constant string), to make sure that the value is correct? otherwise the test seems to repeat the implementation, which doesn't necessarily mean it's really testing :)

Reply:
Agreed, added commit with modified test